apps/prairielearn/src/middlewares/content-security-policy.ts

```diff
@@ -0,0 +1,15 @@
+import { Router } from 'express';
+
+const router = Router();
+
+router.all('/*', function (req, res, next) {
+  // https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP
+  //
+  // Currently, we only use CSP to prevent PrairieLearn from being rendered in
+  // an iframe.
+  res.header('Content-Security-Policy', "frame-ancestors: 'none';");
```

Comment:
What's our thinking on backwards compatibility here? Should we also add `X-Frame-Options: DENY` for older browsers? I think this is what the original security report was actually checking for.

Reply:
It can't hurt. From what I can tell from looking at https://caniuse.com, the main benefit of `X-Frame-Options` is that it's supported by a few more versions of IE.

Reply:
Done in d9b7876.